scranShipper/src/RestaurantMenu/index.jsx

```diff
@@ -1,46 +1,86 @@
 import {useEffect, useState} from "react";
 import './style.css'
+
+import Basket from "../Basket/Basket.jsx";
+
 const RestaurantMenu = (props) => {
+    const [menus, setMenus] = useState([]);
+    const [order, setOrder] = useState({});
```

Comment:
Lots of this has changed to refactor some functions into separate component. Will be a deadly merge into main